index.php

```diff
@@ -1397,8 +1395,6 @@
 				<span id="helioviewer-url-box-msg"></span>
 				<form style="margin-top: 5px;">
 					<input aria-label="Viewport URL" type="text" class="helioviewer-url-input-box helioviewer-url-input-box-video" style="width:98%;" value="https://helioviewer.org" />
-					<label for="helioviewer-url-shorten">Shorten with bit.ly? </label>
-					<input id="helioviewer-url-shorten" type="checkbox" class="helioviewer-url-shorten" />
```

Comment:
Can you add this back and just change the text from "Shorten with bit.ly" to just "Shorten link"